server/http.go

```diff
@@ -525,18 +535,31 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 	userPMISettingPref, err := p.getPMISettingData(user.Id)
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
-		p.askPreferenceForMeeting(user.Id, req.ChannelID)
+		p.askPreferenceForMeeting(user.Id, req.ChannelID, req.RootID)
 		return
 	}
 
 	createMeetingWithPMI := false
 	switch userPMISettingPref {
 	case zoomPMISettingValueAsk:
-		p.askPreferenceForMeeting(user.Id, req.ChannelID)
+		p.askPreferenceForMeeting(user.Id, req.ChannelID, req.RootID)
+
+		if _, err = w.Write([]byte(`{"meeting_url": ""}`)); err != nil {
+			p.API.LogWarn("failed to write the response", "Error", err.Error())
+		}
 		return
 	case "", trueString:
 		createMeetingWithPMI = true
 		meetingID = zoomUser.Pmi
+
+		if meetingID <= 0 {
+			p.postEphemeral(userID, req.ChannelID, req.RootID, "To use Personal Meeting ID (PMI) for creating the meeting, you need to `Enable Personal Meeting ID` from your [zoom settings](https://zoom.us/profile/setting).")
```

Comment:
Can we have this in a constant? A method may be even easier to read, like `p.sendEnableZoomPMISettingMessage(userID, req.ChannelID, req.RootID)`. I trust your judgment on this